docs/configuring_and_running.md

```diff
@@ -60,17 +60,23 @@ Each `[[module]]` section in the `modules.toml` file is responsible for mapping
 The two required directives for a module section are:
 
 - `route`: The path-portion of a URL
-- `module`: The path to the WebAssembly module to execute
+- `module`: A reference to the WebAssembly module to execute
 
 Routes are paths relative to the WAGI HTTP root. Assuming the routes above are running on a server whose domain is `example.com`:
 
 - The `/` route handles traffic to `http://example.com/` (or `https://example.com/`)
 - A route like `/hello` would handle traffic to `http://example.com/hello`
 - The route `/hello/...` is a special wildcard route that handles any traffic to `http://example.com/hello` or a subpath (like `http://example.com/hello/today/is/a/goo/day`)
 
-The `module` directive is a path to a `wasm` or `wat` file on the filesystem.
-We recommend using absolute paths.
-Relative paths will be resolved from the current working directory in which `wagi` was started.
+### Module References
+
+A module reference is a URL. There are three supported module reference schemes:
+
+- `file://`: A path to a `.wasm` or `.wat` file on the filesystem. We recommend using absolute paths beginning with `file://`. Right now, there is legacy support for absolute and relative paths without the `file://` prefix, but we discourge using that. Relative paths will be resolved from the current working directory in which `wagi` was started.
+- `bindle:`: A reference to a Bindle. This will be looked up in the configured Bindle server. Example: `bindle:example.com/foo/bar/1.2.3`.
+- `oci`: A reference to an OCI image in an OCI registry. Example: `oci:foo/bar:1.2.3` (equivalent to the Docker image `foo/bar:1.2.3`).
```

Comment:
It is a bit unclear if modules should be referred to as `oci:<>` or `oci://`.

Reply:
The technical details is that an `oci:` URL that includes a host _can_ use the `//`, but doesn't need to. I think I will just simplify that down to "you should not use //`. Unfortunately, the URL parser has special case logic for `file` that requires `//`.